src/scripts/validation.js

```diff
@@ -80,4 +80,4 @@ const enableValidation = (config) => {
     setEventListeners(formElement, config);
   });
 };
-enableValidation(settings);
+ 
```

Comment:
* Please, export `disableButton` and `resetValidation` functions from `validation.js`, to use them in `index.js`